featurebyte/session/bigquery.py

```diff
@@ -597,30 +617,30 @@ async def get_table_details(
         database_name: str | None = None,
         schema_name: str | None = None,
     ) -> TableDetails:
-        query = (
```

Comment:
Using `INFORMATION_SCHEMA` is not reliable as it will fail when the dataset location does not match the client location

Reply:
Would be good to include this comment in the code.